auth_service/user_management/models/dto.py

```diff
@@ -98,9 +114,33 @@ class Config:  # pylint: disable=missing-class-docstring
         use_enum_values = True
 
 
+class UserAutomaticData(BaseModel):
+    """User data that is automatically created except the ID"""
+
+    registration_date: datetime = Field(default=..., title="Registration date")
+
+    status_change: StatusChange
+
+    class Config:  # pylint: disable=missing-class-docstring
+        frozen = True
+        use_enum_values = True
+
+
+class UserData(UserCreatableData, UserAutomaticData):
+    """User data model without the ID"""
+
+    class Config:  # pylint: disable=missing-class-docstring
+        frozen = True
+        use_enum_values = True
+
+
 class User(UserData):
-    """User"""
+    """Complete user model with ID"""
 
     id: str = Field(  # actually UUID
         default=..., title="ID", description="Internally used ID"
     )
+
+    class Config:  # pylint: disable=missing-class-docstring
+        frozen = True
+        use_enum_values = True
```

Comment:
Again, is inherited. Maybe we can also define a Base class at the top of this module that only adds this config to the BaseModel.

Reply:
Done like that now.